### PR TITLE
Jump to anchor results in scrolling on overflow: hidden

### PR DESCRIPTION
**Pull request by rr-it**
### Description

Jumping to an anchor via link or URL is _the_ programmatic action which triggers scrolling on `overflow: hidden`.

### Motivation

Ever wonder why your `overflow: hidden` jumps around? Better use `overflow: clip`.

### Additional details

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow

### CODE REVIEW

File: files/en-us/web/css/overflow/index.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The `overflow` property is specified as one or two {{CSSXref("overflow_value", "
...
+  - : Overflow content is clipped at the element's padding box. There are no scroll bars, and the clipped content is not visible (i.e., clipped content is hidden), but the content still exists. User agents do not add scroll bars and also do not allow users to view the content outside the clipped region by actions such as dragging on a touch screen or using the scroll wheel on a mouse. The content _can_ be scrolled programmatically (for example, by jumping to an anchor via URL/link or by setting the value of the {{domxref("Element.scrollLeft", "scrollLeft")}} property or the {{domxref("Element.scrollTo", "scrollTo()")}} method), in which case the element box is a scroll container.
```
Comment (estelle) [inline files/en-us/web/css/overflow/index.md:47]:
```suggestion
  - : Overflow content is clipped at the element's padding box. There are no scroll bars, and the clipped content is not visible (i.e., clipped content is hidden), but the content still exists. User agents do not add scroll bars and also do not allow users to view the content outside the clipped region by actions such as dragging on a touch screen or using the scroll wheel on a mouse. The content _can_ be scrolled programmatically (for example, by navigating to a clipped anchor link or by setting the value of the {{domxref("Element.scrollLeft", "scrollLeft")}} property or the {{domxref("Element.scrollTo", "scrollTo()")}} method), in which case the element box is a scroll container.
```
How about this?

**Comment by rr-it** [comment]
See test-case: https://jsfiddle.net/mytv69zd/

**Review by estelle — APPROVED** [review]
Thank you. Also, congratulations on your first merged MDN content PR. Welcome to the team!

**Comment by AbuTariq7x** [comment]
<!DOCTYPE html>
<html lang="ar">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>عرض الوقت والتاريخ والموقع الجغرافي</title>
</head>
<body>
  <h1>الوقت الحالي:</h1>
  <p id="time"></p>
  <h1>التاريخ الحالي:</h1>
  <p id="date"></p>
  <h1>الموقع الجغرافي الحالي:</h1>
  <p id="latitude"></p>
  <p id="longitude"></p>

  <script src="script.js"></script>
</body>
</html>

**Pull request merged**